### PR TITLE
Fix ENV Set Method

### PR DESCRIPTION
**Pull request by RandomNoun7**
Github changed the method of setting environment variables in an action.

https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/

### CODE REVIEW

**Comment by DavidS** [comment]
Similar to https://github.com/puppetlabs/action-litmus_parallel/pull/67 the question here is why the announced (and published) changes by github of the @actions/core API have not addressed this without any changes on our side necessary?

**Comment by RandomNoun7** [comment]
@DavidS Ok, it took me a little while, but I think I understand the question you're asking and I may know the issue. I think dependabot updated the `packages.json` file to reflect the new version of `@actions/core`, but dependabot doesn't know that you then also have to re-vendor all of the packages. When I look at the source code of the actions core library in this repo it doesn't reflect 1.26 and the new method of setting environment variables.

I'm going to update the package and re-push and see if the issue is fixed with no changes at all to our action code.

**Comment by RandomNoun7** [comment]
Ok. So I don't know what weird garbage `jest` was doing that broke everything, but this version here gets the spec tests running just by updating dependencies with no changes to the action code itself.

**Comment by RandomNoun7** [comment]
@DavidS @michaeltlombardi I have split this PR into 3 distinct commits not to help with review.

File: node_modules/@actions/core/lib/core.js [hunk 1]
```diff
@@ -42,8 +42,17 @@ var ExitCode;
  * @param val the value of the variable
  */
 function exportVariable(name, val) {
-    process.env[name] = val;
-    command_1.issueCommand('set-env', { name }, val);
+    const convertedVal = utils_1.toCommandValue(val);
+    process.env[name] = convertedVal;
+    const filePath = process.env['GITHUB_ENV'] || '';
+    if (filePath) {
```
Comment (michaeltlombardi) [inline node_modules/@actions/core/lib/core.js:51]: I suppose this validates that `filePath` is not null by default?

Comment (RandomNoun7) [reply node_modules/@actions/core/lib/core.js:51]: I think it's doing a little bit of error checking to make sure the coding is running in the context of a runner that is updated to use this method of setting env vars. If it's not then it falls back to the old method of using the `set-env` command.

**Review by michaeltlombardi — APPROVED** [review]

**Pull request merged**